lib/logsource.h

```diff
@@ -83,7 +83,8 @@ struct _LogSource
 static inline gboolean
 log_source_free_to_send(LogSource *self)
 {
-  return g_atomic_counter_get(&self->window_size) > 0;
+  return g_atomic_bool_get(&self->forced_suspend) == FALSE
```

Comment:
matter of taste, but as g_atomic_bool_get is boolean, I would use `!g_atomic_bool_get(...)`, provided either setter or getter forcefully translates the counter value to gboolean with !!.